ruby/vampires.rb

```diff
@@ -5,8 +5,14 @@ def interview_test
 puts "Welcome to the interview\nplease be seated!"
 puts "What is your name?"
 person_name = gets.chomp
-puts "How old are you?"
+age = 0 
+until (age > 15 && age <= 80)
```

Comment:
We should be blocking someone from being outside of this age range.  We want to know how old they are or if they are telling the truth.